shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1544,21 +1544,17 @@ - (void)performOrientationUpdate:(UIInterfaceOrientationMask)new_preferences {
           continue;
         }
         UIWindowScene* windowScene = (UIWindowScene*)scene;
-        UIInterfaceOrientationMask currentInterfaceOrientation =
-            1 << windowScene.interfaceOrientation;
-        if (!(_orientationPreferences & currentInterfaceOrientation)) {
-          [self setNeedsUpdateOfSupportedInterfaceOrientations];
-          UIWindowSceneGeometryPreferencesIOS* preference =
-              [[UIWindowSceneGeometryPreferencesIOS alloc]
-                  initWithInterfaceOrientations:_orientationPreferences];
-          [windowScene
-              requestGeometryUpdateWithPreferences:preference
-                                      errorHandler:^(NSError* error) {
-                                        os_log_error(OS_LOG_DEFAULT,
-                                                     "Failed to change device orientation: %@",
-                                                     error);
-                                      }];
-        }
+        UIWindowSceneGeometryPreferencesIOS* preference =
+            [[UIWindowSceneGeometryPreferencesIOS alloc]
+                initWithInterfaceOrientations:_orientationPreferences];
```

Comment:
autorelease]

Reply:
I actually recently caught this on the master branch in Instruments when I was looking at leaks, but I forgot to fix it there.  I know it seems dumb but let's keep this matching https://github.com/flutter/engine/pull/38230 with the leak since that's been baking awhile and we know it doesn't crash.  This is a really tiny leak and I'll fix in master now 🙂

Reply:
im surprised that it doesn't have a `windowSceneGeometryPreferencesIOSWithInterfaceOrientations:` method

Reply:
> This is a really tiny leak and I'll fix in master now

https://github.com/flutter/engine/pull/40957